src/traces/image/defaults.js

```diff
@@ -11,13 +11,19 @@
 var Lib = require('../../lib');
 var attributes = require('./attributes');
 var constants = require('./constants');
+var dataUri = require('../../snapshot/helpers').IMAGE_URL_PREFIX;
 
 module.exports = function supplyDefaults(traceIn, traceOut) {
     function coerce(attr, dflt) {
         return Lib.coerce(traceIn, traceOut, attributes, attr, dflt);
     }
+    coerce('source');
+    if(traceOut.source && !traceOut.source.match(dataUri)) delete traceOut.source;
```

Comment:
Please add a comment that we are sanitizing the `source` here.

Reply:
Done in fe1d2083c4ae61d7c9ee85561fb4654f4cde4643